### ISSUE TITLE
Change background color when click on DevExtreme Reactive Grid row

### ISSUE THREAD

**Issue by veznidav** [DevExpress/devextreme-reactive#1054]
I have a React app which uses DevExtreme's React Grid. I have grouped items same as in this example https://devexpress.github.io/devextreme-reactive/react/grid/demos/featured/virtual-scrolling/. But when I click on group name row, the background color of the row changes for a while. I would like to avoid this background color change but I can't find any CSS that causes this change. Do you have any ideas?

**Comment by gsobolev** [comment]
Hi @veznidav,
 
I'm afraid I can't reproduce the described behavior in the mentioned example. Could you please describe the problem in greater detail or provide a sample demonstrating it?

**Comment by veznidav** [comment]
Of course. This is my Grid: 
```javascript
<Paper style={listStyles}>
	<Grid rows={this.state.gridRowData} columns={listColumns}>
		<GroupingState defaultGrouping={[{ columnName: 'groupName' }]} />
		<IntegratedGrouping />
		<VirtualTable height={kitlistHeight} estimatedRowHeight={70} />
		<TableGroupRow indentColumnWidth={1} />
	</Grid>
</Paper>
```
and this is the behaviour (the blue background only happens on tap on mobile devices, on click there is no background color):

![ezgif com-crop](https://user-images.githubusercontent.com/3127469/39871211-68414bae-5464-11e8-89d2-0c6233238d6d.gif)

**Comment by gsobolev** [comment]
@veznidav, I was able to reproduce this issue only in Google Chrome device emulation mode. However, I did not see similar behavior on real devices.
 
By the way, we do not apply any styling that may cause this visual effect. So I investigated the issue further and this is what I discovered so far. The highlighting appears on a clickable cell that contains a button. This style comes from the Material UI library. I suggest you contact Material UI developers directly to clarify whether this behavior can be changed.
 
Here is a sample that I used during my research: https://codesandbox.io/s/rlykrqkmrp
